rosbag2/include/rosbag2/reader.hpp

```diff
@@ -43,22 +43,17 @@ namespace rosbag2
 class BaseReaderInterface;
 
 /**
- * The SequentialReader allows opening and reading messages of a bag. Messages will be read
- * sequentially according to timestamp.
+ * The Reader allows opening and reading messages of a bag.
```

Comment:
any reason to change this comment? The old version seems more precise?

Reply:
as per discussion